frappe/templates/emails/download_data.html

```diff
@@ -0,0 +1,10 @@
+<p>{{_("Dear {0}").format(user_name)}},</p>
+<p>{{_("We have received a request from you to download your ERPNext data associated with: {0}").format(user)}}.</p>
```

Comment:
same comment related to erpnext

Reply:
changed product name to the hostname (site name).